watertap/examples/flowsheets/mvc/mvc_single_stage_ui.py

```diff
@@ -0,0 +1,649 @@
+        display_units="fraction",
+        rounding=2,
+        description="Utilization factor - [annual use hours/total hours in year]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.TIC,
+        name="Practical investment factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=1,
+        description="Practical investment factor - [total investment cost/direct "
+        "capital costs]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.electricity_cost,
+        name="Electricity cost",
+        ui_units=fs.costing.base_currency / pyunits.kWh,
+        display_units="$/kWh",
+        rounding=3,
+        description="Electricity cost",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.factor_total_investment,
+        name="Total investment factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Utilization factor - [annual use hours/total hours in year]",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.heat_exchanger.material_factor_cost,
+        name="Heat exchanger material cost factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Heat exchanger material cost factor",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.evaporator.material_factor_cost,
+        name="Evaporator material cost factor",
+        ui_units=pyunits.dimensionless,
+        display_units="fraction",
+        rounding=2,
+        description="Evaporator material cost factor",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+    exports.add(
+        obj=fs.costing.compressor.unit_cost,
+        name="Compressor unit cost",
+        ui_units=pyunits.USD_2001,
+        display_units="$(2001)",
+        rounding=2,
+        description="Compressor unit cost",
+        is_input=True,
+        input_category="System costing",
+        is_output=False,
+    )
+
+    # Outlets
+    feed_salinity = 1e3 * fs.feed.properties[0].mass_frac_phase_comp["Liq", "TDS"]
+    exports.add(
+        obj=feed_salinity,
+        name="Feed salinity",
+        ui_units=pyunits.g / pyunits.kg,
+        display_units="g/kg",
+        rounding=2,
+        description="Feed salinity",
+        is_input=False,
+        is_output=True,
+        output_category="Outlets",
+    )
+    brine_salinity = (
+        1e3 * fs.evaporator.properties_brine[0].mass_frac_phase_comp["Liq", "TDS"]
+    )
```

Comment:
Same here regarding units.

Does the `ui_units` argument not do that automatically?

Reply:
I've tested this and ui_units does not do this conversion automatically.